submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -4,134 +4,130 @@
                   Map
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/files.png"  alt="Files icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/files.png" class="app-block__img" alt="Files icon">
                   Files
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/contacts.png"  alt="Contacts icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/contacts.png" class="app-block__img" alt="Contacts icon">
                   Contacts
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/tanslation.png"  alt="Translation icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/tanslation.png" class="app-block__img" alt="Translation icon">
                   Translate
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/videos.png"  alt="Video icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/videos.png" class="app-block__img" alt="Video icon">
                   Videos
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/search.png"  alt="Search icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/search.png" class="app-block__img" alt="Search icon">
                   Search
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/photos.png"  alt="Photos icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/photos.png" class="app-block__img" alt="Photos icon">
                   Photo
                  </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/apps.png"  alt="Apps icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/apps.png" class="app-block__img" alt="Apps icon">
                   Apps
                 </a>
 
-                <a href="#" class="app-cont" tabindex="1">
-                  <img src="img/documents.png"  alt="Documents icon">
+                <a href="#" class="app-block__item">
+                  <img src="img/documents.png" class="app-block__img" alt="Documents icon">
                   Document
                 </a>
               </div>
 
-              <div id="apps-window2">
-                <div id="apps-block2">
+              <div class="apps-window2">
+                <div class="apps-block">
                   
-                  <a href="#" class="app-cont" tabindex="1"> 
-                    <img src="img/chat.png"  alt="Chat icon">
+                  <a href="#" class="app-block__item"> 
```

Comment:
Your parent block is `appS-block` and child elements is `app-block__item`. Is it right?

Reply:
Thank you.
I've fixed it)